job_templates/snippet/publisher_warnings_ng.xml.em

```diff
@@ -1,49 +1,47 @@
 <io.jenkins.plugins.analysis.core.steps.IssuesRecorder plugin="warnings-ng@@7.3.0">
   <analysisTools>
-@[for workspace in ['ws', 'work space']]@
     <io.jenkins.plugins.analysis.warnings.Cmake>
       <id></id>
       <name></name>
-      <pattern>@(workspace)/log/build_*/*/stdout_stderr.log</pattern>
+      <pattern>*/log/build_*/*/stdout_stderr.log</pattern>
```

Comment:
I was thinking something more precise like
```suggestion
      <pattern>{ws,work space}/log/build_*/*/stdout_stderr.log</pattern>
```

I think it only matters if we notice these globs taking a long time to resolve.

Reply:
I don't think that syntax is doing what you intend: [![Build Status](https://ci.ros2.org/buildStatus/icon?job=ci_linux-dirk&build=4)](https://ci.ros2.org/job/ci_linux-dirk/4/) which should have a compiler warning.

> [GNU C Compiler (gcc)] [-ERROR-] No files found for pattern '{ws,work space}/log/build_*/*/stdout_stderr.log'. Configuration error?